EXAMPLES.md

```diff
@@ -222,131 +517,12 @@ The two dependency types are:
-
-![Dependency graph of an application](img/app-cbom.png "Dependency graph of an application")
-
-Key conclusions from the dependency graph:
-- A `uses` path exists from nginx to TLS v1.3, AES-128-GCM, SHA256 and HMAC-DRBG. These crypto assets can be considered used by nginx.
-- No `uses` paths from nginx to TLS v1.2, SSL v3 and MD5 exists. These crypto assets can be considered implemented by libraries (libssl.so, libcrypto.so) but not used by nginx.
-
-Note: Suppose that SSL v3 uses MD5. The dependency graph will still show no `uses` dependency from libcrypto.so to MD5 since there is no `uses` dependency from any component to SSL v3.
-
-The dependency array of the CBOM will look as follows (for simplicity, we use the crypto asset names as the `bom-ref` property. In practice, one use CPE, purl and OID identifiers):
-
-```
-    "dependencies": [
-        {
-            "ref": "nginx",
-            "dependsOn": [
-                "libssl"
-            ],
-            "dependencyType": "uses"
-        },
-        {
-            "ref": "libssl.so",
-            "dependsOn": [
-                "TLS v1.3", "TLS v1.2", "SSL v3"
-            ],
-            "dependencyType": "implements"
-        },
-        {
-            "ref": "libssl.so",
-            "dependsOn": [
-                "TLS v1.3"
-            ],
-            "dependencyType": "uses"
-        },
-        {
-            "ref": "TLS v1.3",
-            "dependsOn": [
-                "libcrypto.so"
-            ],
-            "dependencyType": "uses"
-        },
-        {
-            "ref": "TLS v1.2",
-            "dependsOn": [
-                "libcrypto.so"
-            ],
-            "dependencyType": "uses"
-        },
-        {
-            "ref": "SSL v3",
-            "dependsOn": [
-                "libcrypto.so"
-            ],
-            "dependencyType": "uses"
-        },
-        {
-            "ref": "libcrypto.so",
-            "dependsOn": [
-                "MD5", "AES-128-GCM", "SHA256", "HMAC-DRBG"
-            ],
-            "dependencyType": "implements"
-        },
-        {
-            "ref": "libcrypto.so",
-            "dependsOn": [
-                "AES-128-GCM", "SHA256", "HMAC-DRBG"
-            ],
-            "dependencyType": "uses"
-        }
-    ]
-```
-
-## Dependencies viewed from a library
-
-The chart below shows a partial dependency graph from a CBOM of crypto library `libssl.so`.
+> **Note:** In classical SBOM, a component represents an external dependency to a library, application, service, etc. Since for some crypto, e.g. the relationship between a protocol and the algorithms used, this relationship is not external but internal, we decided to express this relationship differently by using specific properties to reference assets rather than using the classical dependency implementation.
```

Comment:
```suggestion
> **Note:** In classic SBOM, a component represents a library, application, service, etc. that is used as a dependency within the application. Since for some crypto, e.g. the relationship between a protocol and the algorithms used, this relationship is not external but internal, we decided to express this relationship differently by using specific properties to reference assets rather than using the classical dependency implementation.
```